src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4decoder/common/stream/monitoringmetric/SRTMetric.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.x4decoder.common.stream.monitoringmetric;
+
+/**
+ * Set of SRT monitoring metrics keys
+ *
+ * @author Harry
+ * @since 1.0
+ */
+public enum SRTMetric {
+
+	// Static metric
+	ENCRYPTION("SRT Encryption"),
+	KEY_LENGTH("SRT KeyLength"),
+	DECRYPT_STATE("SRT DecryptState"),
+	LOST_PACKETS("SRT LostPackets"),
+	SENT_ACKS("SRT SentAcks"),
+	SENT_NAKS("SRT SentNaks"),
+	PATH_MAX_BANDWIDTH("SRT PathMaxBandwidth"),
+	RTT("SRT Rtt"),
+
+	// To Do: Can be Historical metric in next version
```

Comment:
the syntax is TODO 
you should use the correct syntax for the tools can scan it

Reply:
please check the other places as well

Reply:
I fixed it